libs/components/lists/testing/src/modules/list-summary/list-summary-harness.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { provideNoopAnimations } from '@angular/platform-browser/animations';
+import { SkyListSummaryModule } from '@skyux/lists';
+
+import { SkyListSummaryHarness } from './list-summary-harness';
+import { SkyListSummaryItemHarness } from './list-summary-item-harness';
+
+@Component({
+  imports: [SkyListSummaryModule],
+  template: `
+    <sky-list-summary data-sky-id="test-summary">
+      <sky-list-summary-item
+        data-sky-id="test-item-1"
+        [label]="'Total Items'"
+        [value]="42"
+      />
+      <sky-list-summary-item
+        data-sky-id="test-item-2"
+        [label]="'Average Score'"
+        [value]="'98.5%'"
+      />
+    </sky-list-summary>
+  `,
+})
+class TestComponent {}
+
+@Component({
+  imports: [SkyListSummaryModule],
+  template: `
+    <sky-list-summary data-sky-id="test-summary-with-help">
+      <sky-list-summary-item
+        data-sky-id="test-item-with-help"
+        [label]="'Revenue'"
+        [value]="1000000"
+        [helpPopoverContent]="
+          'This represents the total revenue for the current period.'
+        "
+        [helpPopoverTitle]="'Revenue Help'"
```

Comment:
```suggestion
        [helpPopoverTitle]="'Revenue help'"
```
Not sure if this title is really necessary at all.

Reply:
this is just a test component, the content isn't exposed anywhere. it's used to validate that the popover title is exposed to the harness